sensor_filter:
  ros__parameters:
    wheelbase: 0.258
    wheelradius: 0.065

    max_speed: 3.0
    min_speed: -0.1
    max_acceleration: 6.0
    min_acceleration: -10.0
    max_delta: 0.314159
    min_delta: -0.314159

    x_linear_acceleration_offset: -0.2
    z_angular_velocity_offset: 0.009